### PR TITLE
Fix query pool size

### PR DESCRIPTION
**Pull request by raphlinus**
This was causing harmless warnings on mac but causing rendering to fail on Pixel 6.

### CODE REVIEW

**Comment by dfrg** [comment]
While this is in PR form, should we just make this an associated constant... Renderer::QUERY_POOL_SIZE?

File: piet-gpu/bin/cli.rs [hunk 1]
```diff
@@ -232,7 +232,7 @@ fn main() -> Result<(), Error> {
         let session = Session::new(device);
 
         let mut cmd_buf = session.cmd_buf()?;
-        let query_pool = session.create_query_pool(8)?;
+        let query_pool = session.create_query_pool(12)?;
```
Comment (dfrg) [inline piet-gpu/bin/cli.rs:235]:
```suggestion
        let query_pool = session.create_query_pool(Renderer::QUERY_POOL_SIZE)?;
```

Comment (dfrg) [reply piet-gpu/bin/cli.rs:235]: missed one :)

Comment (raphlinus) [reply piet-gpu/bin/cli.rs:235]: Gah, I had it in my buffer, just didn't save to the PR. Good catch thanks.

**Review by dfrg — APPROVED** [review]
lgtm!

**Pull request merged**